### PR TITLE
Create fetch rule for live update

### PR DESCRIPTION
**Pull request by yiqunw700**
Towards: #1028
Towards: #1801

When working on pr #1800 to create a separate shader test for `CustomBrushTest`, I would like to disable it from the `RenderAllExamples` test but then realized that it would not fetch the dcf file with the gradle task. So I created the fetch rule and made the `ShaderTest` and `StateCustomizationsUnitTest` fetchable in this pr.

Also explored a little bit further to use this fetch rule to make `DesignSwitcherBasicTests` able to fetch dcf file.

Tested locally by running `./gradlew fetchAndUpdateFigmaFiles` and `./gradlew recordRoborazziDebug`

### CODE REVIEW

File: designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt [hunk 1]
```diff
@@ -298,15 +299,26 @@ internal fun designSwitcherDocName() = "DesignSwitcherDoc"
...
-private fun getLiveMode(): LiveUpdateMode {
-    return if (DISABLE_LIVE_MODE) {
-        LiveUpdateMode.OFFLINE
+private var enableLiveModeForTesting = false
+
+internal fun enableLiveModeForTesting(enabled: Boolean) {
+    if (Build.FINGERPRINT == "robolectric") {
+        enableLiveModeForTesting = enabled
     } else {
+        // TODO: no exception for integration tests...
```
Comment (timothyfroehlich) [inline designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt:312]: Clean up the Todo? I think it's fine as is, it's certainly not an error that should be handled.

**Review by timothyfroehlich — APPROVED** [review]
Looks great! Big fan of this change.

**Comment by rylin8** [comment]
Is the removal of the custom brush test because of the separate shader test added previously?

File: designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt (hunk 1 above)
Comment (yiqunw700) [reply designcompose/src/main/java/com/android/designcompose/DesignSwitcher.kt]: Done.

**Comment by yiqunw700** [comment]
> Is the removal of the custom brush test because of the separate shader test added previously?

Yes.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**